.gitignore

```diff
@@ -1,2 +1,4 @@
 dist/
 tmp/
+bin/
```

Comment:
Why `bin`, if we have `dist/`?

Reply:
Ah, this is actually my personal habit, I shouldn't submit it.

Reply:
I sometimes do this, either this or `dist/`. Either way, it doesn't fit within this PR.